test/quantization/test_quantized_op.py

```diff
@@ -456,6 +456,43 @@ def test_qclamp(self, X, min_val, max_val):
             qY_hat = op(qX, min_val, max_val)
             self.assertEqual(qY, qY_hat, message="{} qclamp failed".format(name))
 
+    """Tests the correctness of the quantized::clamp_with_tensors op."""
+    @given(X=hu.tensor(shapes=hu.array_shapes(1, 8, 1, 8),
+                       elements=hu.floats(-1e6, 1e6, allow_nan=False),
+                       qparams=hu.qparams()))
+    def test_qclamp_with_tensors(self, X):
```

Comment:
Maybe add a tests where an error is expected to be thrown, s.a. incorrect shape, incorrect quant params, etc.

Reply:
Done (as in above comment).